classic-editor.php

```diff
@@ -331,12 +334,11 @@ public static function notice_after_upgrade() {
 			return;
 		}
 
-		$message = __( 'The Classic Editor plugin prevents use of the new Block Editor.', 'classic-editor' );
-
+		$message = __( 'Classic Editor plugin inhibits use of the new Block Editor if it is not properly configured.', 'classic-editor' ); # TWEAK: 1 - changed the "catastrophic" term "prevents" with the more "friendly-confidential" term "inhibit" 2 - more understanding anc concise description 3 - separation in a new line generic part of the warning with the specific setting action 4 - make the warning compatible with small screen size

```

Comment:
Hmm, that's not really the case. Classic Editor can be configured to always replace the Block Editor.